contrib/jira_renderer.py

```diff
@@ -63,10 +63,14 @@ def render_image(self, token):
         return template.format(src=token.src)
 
     def render_link(self, token):
-        template = '[{inner}|{target}]'
```

Comment:
OK, this seems to be in-line with the existing code here.

Yet, while being at it, what about also swapping the order of creating and passing values to the `template`, so that it is more natural (like inside the `render_block_code()` method)?

Reply:
@pbodnar: sure, no problem, I was just trying to follow the style and minimize the diff, I'll amend and force-push that one.

Reply:
I see, thanks for the amendment.